### PR TITLE
Add Grey Baker to the team

### PR DESCRIPTION
**Pull request by indirect**

### CODE REVIEW

**Comment by greysteil** [comment]
👋

File: lib/tasks/contributors.rake [hunk 1]
```diff
@@ -5,7 +5,7 @@ namespace :contributors do
       [url, "s=40"].join(connector)
     end
 
-    core_team = %w[indirect segiddins hsbt colby-swandale]
```
Comment (segiddins) [inline lib/tasks/contributors.rake]: this seems to be missing grey ;)

Comment (indirect) [reply lib/tasks/contributors.rake]: oh no

**Comment by indirect** [comment]
@bundlerbot r+

**Pull request merged**